accounts_test.go

```diff
@@ -670,7 +671,8 @@ func TestRemoveAccountKey(t *testing.T) {
 	result, err = b.ExecuteNextTransaction()
 	assert.NoError(t, err)
 
-	unittest.AssertFVMErrorType(t, &fvmerrors.InvalidProposalSignatureError{}, result.Error)
+	var sigErr *fvmerrors.InvalidProposalSignatureError
+	assert.True(t, errors.As(result.Error, &sigErr))
```

Comment:
Maybe use the `assert.ErrorAs` assertion here, as it provides a better description when the assertion fails. Same in the other tests below